stable/cluster-autoscaler/templates/podsecuritypolicy.yaml

```diff
@@ -1,26 +1,42 @@
 {{- if .Values.rbac.pspEnabled }}
```

Comment:
Will you please document the PSP in the README (I've just noticed it's entirely absent!), and make a brief note that this PR takes quite a departure from the previous versions?